lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -68,12 +66,10 @@ protected function transform($item) {
 
     $campaign = (object) $item->campaign;
     $current_run = $campaign->campaign_runs['current']['en']['id'];
-
     $current = ($item->campaign_run == $current_run);
-
     $data += $this->transformSignup($item, $current);
-
     $data['campaign'] = $this->transformCampaign((object) $item->campaign);
+    $data['reportback'] = $this->transformReportback((object) $item->rbid);
```

Comment:
update the `$item->rbid` here too I think... should be `$item->reportback` now, right?